docs/source/tutorial/basic.rst

```diff
@@ -359,14 +359,17 @@ The state of an optimizer can also be saved by the same functions:
 
 .. doctest::
 
-   >>> serializers.save_hdf5('my.state', optimizer)
-   >>> serializers.load_hdf5('my.state', optimizer)
+   >>> serializers.save_npz('my.state', optimizer)
+   >>> serializers.load_npz('my.state', optimizer)
 
 .. note::
    Note that serialization of optimizer only saves its internal states including number of iterations, momentum vectors of MomentumSGD, etc.
    It does not save the parameters and persistent values of the target link.
    We have to explicitly save the target link with the optimizer to resume the optimization from saved states.
 
+Support of the HDF5 format is enabled if the h5py package is installed.
+Serialization and deserialization with the HDF5 format are almost identical to those with the NPZ format;
+just replace ``save_npz`` and ``load_npz`` by :func:`~serializers.save_hdf5` and :func:`~serializers.load_hdf5`, respectively.
```

Comment:
Use :func:`~serializers.save_npz` and :func:`~serializers.load_npz` respectively.